recipes/mysql-connector-c/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "mysql-connector-c" %}
+{% set version = "6.1.6" %}
+{% set sha256 = "2222433012c415871958b61bc4f3683e1ebe77e3389f698b267058c12533ea78" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://ftp.iij.ad.jp/pub/db/mysql/Downloads/Connector-C/{{ name|lower }}-{{ version }}-src.tar.gz
```

Comment:
My fault. It is the same file and I was updating the url when I was looking at win source version but forgot to add it. I will update the url and also take another look at the windows version.